.autover/changes/7db9a194-0cc2-43e7-81da-e2f468d3fc90.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Projects": [
+    {
+      "Name": "Amazon.Lambda.AspNetCoreServer",
+      "Type": "Patch",
+      "ChangelogMessages": [
+        "Fixed issue \u0027(EmptyBodyBehavior = EmptyBodyBehavior.Allow)\u0027 not being honored when request body was empty"
```

Comment:
Does autover require 2 files? Also curious if `\u0027` is here intentionally.

Reply:
I think if I crafted the tool by hand I could have done it with one file with multiple items in the `Projects` array. I used the CLI which just does one project at a time. For this change the change log will end up the same whether I did it as one file or 2.

As for the `\u0027` I think that was just the escaping of the `'` I used on the CLI.